assets/src/dashboard/app/views/myStories/index.js

```diff
@@ -128,21 +165,36 @@ function MyStories() {
         layoutStyle={viewStyle}
         onPress={handleViewStyleBarButtonSelected}
       />
-      <StoryGrid>
-        {filteredStories.map((story) => (
-          <CardGridItem key={story.id}>
-            <CardPreviewContainer
-              onOpenInEditorClick={() => {}}
-              onPreviewClick={() => {}}
-              previewSource={'http://placeimg.com/225/400/nature'}
-            />
-            <CardTitle
-              title={story.title}
-              modifiedDate={story.modified.startOf('day').fromNow()}
-            />
-          </CardGridItem>
-        ))}
-      </StoryGrid>
+      <FontProvider>
+        <TransformProvider>
+          <UnitsProvider pageSize={pageSize}>
+            <StoryGrid>
+              {filteredStories.map((story) => (
+                <CardGridItem key={story.id}>
+                  <CardPreviewContainer
+                    onOpenInEditorClick={() => {}}
+                    onPreviewClick={() => {}}
+                  >
+                    {story.pages[0].elements.map(({ id, ...rest }) => {
```

Comment:
Let's wrap this info a `PreviewPage` or something component right away. You can keep it locally for now. But that's something I'd like to re-add on our side as well to make the structure follow data model cleaner.